pkg/api/validation/validation_test.go

```diff
@@ -5441,6 +5440,27 @@ func TestValidateNode(t *testing.T) {
 				ExternalID: "external",
 			},
 		},
+		"duplicated-taints-with-same-key-effect": {
+			ObjectMeta: api.ObjectMeta{
+				Name: "dedicated-node1",
+				// Add a taint with an empty key to a node
```

Comment:
This comment should say something like "Add two taints to the node with the same key and effect; should be rejected."